src/components/atoms/link.tsx

```diff
@@ -14,7 +24,15 @@ export const Link = ({ ariaLabel, children, className, disabled, target, title,
 	);
 
 	return (
-		<LinkReact aria-label={ariaLabel} className={linkClass} id={id} target={target} title={title} to={to}>
+		<LinkReact
```

Comment:
Hey, adding `onClick` to react-router-dom Links is problematic because:
1. Breaks browser expectations - disrupts right-click, middle-click, and keyboard navigation patterns
2. Accessibility issues - screen readers expect links to navigate, not perform actions
SEO/URL problems - can prevent proper navigation and browser history management
3. Better alternatives exist - use useNavigate() for programmatic navigation or separate Button components for actions
4. Mixing concerns - Links should navigate, Buttons should perform actions - keep them semantically separate

In our case, you should simply `triggerEvent(EventListenerName.closeChatBotOnLogo);` and that's it.